docs/contributing/semgrep-core-contributing.md

```diff
@@ -146,6 +146,85 @@ code .
 
 ## Testing performance
 
+### Exploring results from a slow run of Semgrep
+
+#### Interpreting the result object
+
+For full timing information, run Semgrep with `--time` and `--json`. In addition, you will want to `time` the entire command to get the true wall time. The `--json` argument produces a large amount of output, so redirecting the output to a file with `-o` is recommended. The full command would look sometthing like:
```

Comment:
Idk if you git blamed these but these are some of the docs that have been around longer than I have :D As such the original writing predated our establishment of standards. I will just make a few rewrites.

```suggestion
For full timing information, run Semgrep with `--time` and `--json` flags. In addition, you can add `time` at the beginning of the command to get the true wall time. The `--json` argument produces a large amount of output, so redirecting the output to a file with `-o` is recommended. 

See the following example for the full command:
```